Fix #91. Make all pointer events composed events

hayatoito opened the pull request:
All pointer events should propagate across Shadow DOM bounaries.
See https://github.com/w3c/webcomponents/issues/513 for details.

hayatoito committed:
Fix #91. Make all pointer events composed events

All pointer events should propagate across Shadow DOM bounaries.
See https://github.com/w3c/webcomponents/issues/513 for details.

patrickhlauke commented:
as I'm not completely au-fait with what composed events actually are/do, nor if there are any issues with a W3C spec referencing WHATWG (political or otherwise) ... @RByers, thoughts?

RByers commented:
I suggest we wait until UIEvents [has been updated](https://github.com/w3c/uievents/issues/89) for this before trying to review a PR for PointerEvents (since we mostly just copy the patterns from UI Events here).

hayatoito commented:
@RByers

PR for UI events has been merged w3c/uievents#91.
Could you have a chance to review this PR for Pointer Events?

RByers commented on index.html:
```diff
@@ -393,7 +394,7 @@
                 <p>To <dfn>fire a pointer event name e</dfn> means to <dfn>fire an event named e</dfn> as defined in [[!DOM4]] with an event using the <a>PointerEvent</a> interface whose attributes are set as defined in <a href="#pointerevent-interface"><code>PointerEvent</code> Interface</a>.</p>
                 
-                <p>Initialize the <code>bubbles</code> attribute for the event to <code>true</code> if the event name is</p>
+                <p>Initialize the <code>bubbles</code> attribute and the <code>composed</code> [[!WHATWG-DOM]] attribute for the event to <code>true</code> if the event name is</p>
```
The below list doesn't include `pointerenter` or `pointerleave` but you want those to be composed, right?  I suggest moving the text for composed up to the above sentence so that it applies to all pointer events.

hayatoito replied:
Good catch! Done.

RByers commented:
Oh, this is non-trivial enough that you should probably also add an entry to the [revision history](https://w3c.github.io/pointerevents/#revision-history).

hayatoito commented:
> Oh, this is non-trivial enough that you should probably also add an entry to the revision history.

Thanks! I have added an entry to the revision history.

I have squashed the commits. Could you review PR again?

patrickhlauke commented:
LGTM
